.github/workflows/ci-build.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         run: yarn install
       - name: lint
         run: yarn lint:js
+      - name: validate
+        run: yarn tsc --noEmit
```

Comment:
We already have a script for this `yarn lint:ts`

https://github.com/emberjs/ember-test-waiters/blob/1f4ed4d96c4e5958c4ab286f734ee052fd60e236/package.json#L20-L31

We should tweak to match the more modern addon blueprint for our scripts, so we run `yarn lint` in CI and that runs `lint:*` (along with `yarn test` running `yarn lint`, to ensure that local testers just running `yarn test` have a proper "pass"/"fail" experience).

Reply:
Oh good call. I'll update to that.

Reply:
OK, I've updated the scripts to more closely match the blueprint, though there are some small differences. Let me know if you think that makes sense.